UnityProject/Assets/Mirror/Runtime/NetworkServer.cs

```diff
@@ -70,6 +70,10 @@ static void Initialize()
             if (initialized)
                 return;
 
+            //CUSTOM UNITYSTATION CODE//
+            //So it can handle Not having domain reloading on
+            isLoadingScene = false;
```

Comment:
so we force the scene to be flagged as not loading when initializing the server?

Reply:
it should be set to true since this should run before that

Reply:
ELI5 how does it affect scene loading?

Reply:
It doesn't